Libraries/Text/RCTTextFieldManager.m

```diff
@@ -39,8 +39,16 @@ - (BOOL)textField:(RCTTextField *)textField shouldChangeCharactersInRange:(NSRan
   }
 
   if (textField.maxLength == nil || [string isEqualToString:@"\n"]) {  // Make sure forms can be submitted via return
+    if (textField.autocorrectionType == UITextAutocorrectionTypeYes) {
```

Comment:
'Are there no other callbacks we get after the auto correct suggestion gets committed?'
I don't think so.